state/upgrades.go

```diff
@@ -235,7 +235,7 @@ func DropOldLogIndex(st *State) error {
 	// If the log collection still has the old e,t index, remove it.
 	key := []string{"e", "t"}
 	db := st.MongoSession().DB(logsDB)
-	collection := db.C(logsC)
+	collection := db.C("logs")
```

Comment:
It's used in steps_21 - although I guess if someone is upgrading to 2.1 they aren't running this binary, are they? So the then only use for this is if someone is upgrading from 2.0 to 2.2? You're right - I'll delete the step. It feels slightly weird to be deleting something from the 2.1 steps because of something we're changing in 2.2 though.